tapdb/sqlc/queries/transfers.sql

```diff
@@ -23,11 +23,36 @@ INSERT INTO asset_transfer_outputs (
     amount, serialized_witnesses, split_commitment_root_hash,
     split_commitment_root_value, proof_suffix, num_passive_assets,
     output_type, proof_courier_addr, asset_version, lock_time,
-    relative_lock_time
+    relative_lock_time, proof_deliver_complete
 ) VALUES (
-    $1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15
+    $1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15, $16
 );
 
+-- name: SetTransferOutputProofDeliveryStatus :exec
+WITH target(output_id) AS (
+    SELECT output_id
+    FROM asset_transfer_outputs output
+    JOIN asset_transfers
+      ON output.transfer_id = asset_transfers.id
+    JOIN chain_txns
+      ON asset_transfers.anchor_txn_id = chain_txns.txn_id
+    JOIN script_keys
+      ON output.script_key = script_keys.script_key_id
+    WHERE (chain_txns.txid = sqlc.narg('anchor_tx_id') OR sqlc.narg('anchor_tx_id') IS NULL)
```

Comment:
Was thinking that we may not need every field here, but after finishing the review that seems to make sense / we don't actually have to use them all when using this query.